LumpNav/LumpNav.py

```diff
@@ -706,6 +762,45 @@ def setupNavigationPanel(self):
     self.deleteLastFiducialDuringNavigationButton.setEnabled(False)
     self.contourAdjustmentFormLayout.addRow(self.deleteLastFiducialDuringNavigationButton)
 
+    # Distance Collapsible
+    self.distanceCollapsibleButton = ctk.ctkCollapsibleGroupBox()
+    self.distanceCollapsibleButton.title = "Distance"
+    self.distanceCollapsibleButton.collapsed=True
+    self.navigationCollapsibleLayout.addRow(self.distanceCollapsibleButton)
+
+    # Layout within the collapsible button
+    self.distanceFormLayout = qt.QFormLayout(self.distanceCollapsibleButton)        
+
+    self.activateDistanceButton = qt.QPushButton("Activate")
+    setButtonStyle(self.activateDistanceButton)
+    self.activateDistanceButton.setCheckable(True)
+    self.distanceFormLayout.addRow(self.activateDistanceButton)
+    
+    self.distanceComboBoxLabel = qt.QLabel('Text location:')

```

Comment:
This is too much detail on the GUI. Would it be possible to figure this out automatically? Or just make it configurable only through the .ini file?